src/client.js

```diff
@@ -40,6 +40,15 @@ class Client extends Api {
 		} );
 	}
 
+	// Override base API to keep from loading a temp table on different connection.
+	async bulkLoad( tableName, options ) {
+		if ( tableName.indexOf( "#" ) >= 0 ) {
```

Comment:
Sorry I just noticed this. I believe a permanent table can have a `#` in its name, so this should be `===`.

Reply:
@calvinb cool, I'll look into this. I bet I need to split on dots too?

Reply:
Good idea.

Reply:
Fixed! Let me know how that looks.